src/backend/distributed/relay/relay_event_utility.c

```diff
@@ -511,6 +571,19 @@ AppendShardIdToConstraintName(AlterTableCmd *command, uint64 shardId)
 	}
 }
 
```

Comment:
Very minor. We need two newlines between functions. You can catch these typo of errors with citus_indent :)